packages/@expo/cli/src/start/server/metro/inspector-proxy/messages/VscodeCompat.ts

```diff
@@ -0,0 +1,67 @@
+import type { Protocol } from 'devtools-protocol';
```

Comment:
Assuming this is in devDependencies already?

Reply:
Yep! See: https://github.com/expo/expo/pull/21449/files#diff-00b4bb0f6ea42a3d74201ea46359ab71c5085d6a17f80894dfc24d8614420018R135